package.json

```diff
@@ -26,6 +26,7 @@
     "@shopgate/pwa-common": "^0.1.1",
     "@shopgate/pwa-common-commerce": "^0.1.1",
     "@shopgate/pwa-core": "^0.1.2",
+    "@shopgate/pwa-webcheckout-shopify": "github:shopgate/pwa-webcheckout-shopify",
```

Comment:
This needs to be changed to an npm version before the pull request can be merged.